lib/util/git/auth.ts

```diff
@@ -151,15 +161,15 @@ export function getGitEnvironmentVariables(
   let environmentVariables: NodeJS.ProcessEnv = {};
 
   // hard-coded logic to use authentication for github.com based on the githubToken for api.github.com
-  const githubToken = find({
+  const gitHubHostRule = find({
```

Comment:
Is this rename required?

Reply:
It isn't required, but while looking through it I noticed that `gitHubHostRule` reflects the purpose better than `githubToken`.

Reply:
should I move this change in a separate PR?